src/main/resources/org/jenkinsci/plugins/zap/ZAPDriver/config.jelly

```diff
@@ -115,6 +115,13 @@ SOFTWARE.
     </f:entry>
 
     <f:entry>
+
+      <f:optionalBlock title="${%Context Alert Filters}" field="loadAlertsFilter" inline="true">
```

Comment:
Rename to `Alert Filters` from `Context Alert Filters`. (it's implied) since it's now nested under the context configuration. :)